parameter_files/fates_params_default.cdl

```diff
@@ -244,6 +271,15 @@ variables:
 	double fates_hydr_thetas_node(fates_hydr_organs, fates_pft) ;
 		fates_hydr_thetas_node:units = "cm3/cm3" ;
 		fates_hydr_thetas_node:long_name = "saturated water content" ;
+	double fates_hydr_vg_alpha_node(fates_hydr_organs, fates_pft) ;
```

Comment:
Do we have a way of indicating in that you only need to define these if using van Genuchten mode? I can imagine it freaking out plant trait measuring folk innocently browsing through the file!

Reply:
updated text to this effect